### PR TITLE
Fix Memory Leak

### PR DESCRIPTION
**Pull request by usta**
include_paths was leaking

### CODE REVIEW

File: src/sass_context.cpp [hunk 1]
```diff
@@ -220,7 +220,10 @@ extern "C" {
       size_t imp_size = 0; while (imp) { imp_size ++; imp = imp->next; }
       // create char* array to hold all paths plus null terminator
       const char** plugin_paths = (const char**) calloc(imp_size + 1, sizeof(char*));
-      if (plugin_paths == 0) throw(std::bad_alloc());
+      if (plugin_paths == 0){
```
Comment (xzyfer) [inline src/sass_context.cpp]: Please follow the existing code-style and indentation.

```
if (plugin_paths == 0) {
		  free(include_paths); //free include_paths before throw
		  throw(std::bad_alloc());
}
```

Comment (xzyfer) [inline src/sass_context.cpp]: Also spaces, not tabs.

**Comment by xzyfer** [comment]
Thanks @usta but please follow the existing code-style.

**Comment by usta** [comment]
I hope  this one is correct code-style

**Comment by xzyfer** [comment]
Much better, thank you.

**Comment by usta** [comment]
you are welcome

**Pull request merged**